src/test/java/io/logz/guice/jersey/JettyFilterRegistrationTest.java

```diff
@@ -0,0 +1,50 @@
+package io.logz.guice.jersey;
+
+import io.logz.guice.jersey.configuration.JerseyConfiguration;
+import io.logz.guice.jersey.configuration.JerseyConfigurationBuilder;
+import io.logz.guice.jersey.filters.JettyTestFilter;
+import io.logz.guice.jersey.resources.TestResource;
+import io.logz.guice.jersey.supplier.JerseyServerSupplier;
+import me.alexpanov.net.FreePortFinder;
+import org.eclipse.jetty.server.Server;
+import org.eclipse.jetty.servlet.FilterHolder;
+import org.glassfish.jersey.server.ResourceConfig;
+import org.junit.Test;
+
+import javax.servlet.DispatcherType;
+import javax.ws.rs.core.Response;
+import java.util.EnumSet;
+import java.util.HashMap;
+import java.util.Map;
+
+import static org.junit.Assert.assertEquals;
+
+public class JettyFilterRegistrationTest {
+
+    @Test
+    public void testJettyFilter() throws Exception {
+        ResourceConfig resourceConfig = new ResourceConfig().registerClasses(TestResource.class);
+        JerseyConfigurationBuilder configurationBuilder = JerseyConfiguration.builder()
+                .withResourceConfig(resourceConfig);
```

Comment:
Maybe use like other tests
```
                .addResourceClass(TestResource.class)
                .registerClasses(TestFilter.class);
```

Reply:
Can't use `.registerClasses` since this just allows registering resources on Jersey and not `Filter` or others directly to Jetty / WebAppContext

Reply:
I now see you have changed it 👍